src/createTippy.ts

```diff
@@ -830,9 +822,9 @@ export default function createTippy(
       ...options,
       ignoreAttributes: true,
     })
-    nextProps.ignoreAttributes = hasOwnProperty(options, 'ignoreAttributes')
+    nextProps.ignoreAttributes = (hasOwnProperty(options, 'ignoreAttributes')
       ? options.ignoreAttributes
-      : prevProps.ignoreAttributes
+      : prevProps.ignoreAttributes) as boolean
```

Comment:
I think `hasOwnProperty` could also check for `undefined`, so it falls back to `prevProps` in this case.

```ts
const options = { ignoreAttributes: undefined }
hasOwnProperty(options, 'ignoreAttributes') // true
```

If it was `false` then we could write `options.ignoreAttributes!` in the ternary and skip `as boolean` assignment.